src/main/java/com/dev/cinema/models/MovieSession.java

```diff
@@ -0,0 +1,62 @@
+package com.dev.cinema.models;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "movie_session")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @Fetch(FetchMode.JOIN)
+    private Movie movie;
+    @ManyToOne
+    @Fetch(FetchMode.JOIN)
```

Comment:
What's the difference between **_@Fetch(FetchMode.JOIN)_** and **_fetch type EAGER_**?